src/index.html

```diff
@@ -1,18 +1,22 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Lokalkauf</title>
+    <base href="/" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="icon" type="image/x-icon" href="favicon.ico" />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
```

Comment:
I think this is a strange formatting rule. Can we disable this?

Reply:
what exactly do you mean? We could increase the max-line-length so it doesn't break in that case. I think thats whats causing it.